synfig-core/src/synfig/valuenodes/valuenode_modulo.cpp

```diff
@@ -0,0 +1,239 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file valuenode_modulo.cpp
+**	\brief Implementation of the "Modulo" valuenode conversion.
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**	Copyright (c) 2022 Synfig Contributors
+**
+**	This file is part of Synfig.
+**
+**	Synfig is free software: you can redistribute it and/or modify
+**	it under the terms of the GNU General Public License as published by
+**	the Free Software Foundation, either version 2 of the License, or
+**	(at your option) any later version.
+**
+**	Synfig is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+**	GNU General Public License for more details.
+**
+**	You should have received a copy of the GNU General Public License
+**	along with Synfig.  If not, see <https://www.gnu.org/licenses/>.
+**	\endlegal
+*/
+/* ========================================================================= */
+
+/* === H E A D E R S ======================================================= */
+
+#ifdef USING_PCH
+#	include "pch.h"
+#else
+#ifdef HAVE_CONFIG_H
+#	include <config.h>
+#endif
+
+#include "valuenode_modulo.h"
+#include "valuenode_const.h"
+
+#include <synfig/general.h>
+#include <synfig/localization.h>
+#include <synfig/valuenode_registry.h>
+#include <synfig/vector.h>
+#include <synfig/angle.h>
+#include <synfig/real.h>
+
+#include <ETL/misc>
+#include <ETL/stringf>
+
+#include <stdexcept>
+
+#endif
+
+/* === U S I N G =========================================================== */
+
+using namespace etl;
+using namespace synfig;
+
+/* === M A C R O S ========================================================= */
+
+/* === G L O B A L S ======================================================= */
+
+REGISTER_VALUENODE(ValueNode_Modulo, RELEASE_VERSION_1_4_0, "modulo", N_("Modulo"))
```

Comment:
I'm hesitant with this version detail: should we create RELEASE_VERSION_1_4_0?

Reply:
Do you mean `1_4_3`?

Reply:
hmmm Maybe 1_6_0?

Reply:
> hmmm Maybe 1_6_0?

Agree, it makes sense. 1.4.x is mostly bug fixes, so 1.6.0 is much better for a new feature.